pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.black]
+line-length = 140
+target-version = ['py37']
+skip-string-normalization = true
```

Comment:
I don’t know whether you discussed it elsewhere, but I’m in favor to remove this option. Even if my current personal style is using single quotes (where I’m debating to switch this over), I think it would be worth to enable string normalization, because:

- black introduces enough changes for our code, so it would be the perfect time – even if it changes some more lines
- it correctly handles tricky cases like `f'Look at ma map, it calls itself \'mappy\': {look["here"]}' and gives one thing less to debate over in reviews
- As a more idealistic argument, it feels like some “configuration through the backdoor” when we keep this option

Reply:
We didn't discuss that yet. You kind of have a point that we're changing everything anyway, so we might as well change this. And it does fix one more thing we quibble about in reviews. So I'm slightly in favor. What do the others think?

Reply:
hit it! (i'm in favor of normalizing the strings as well)

Reply:
Also in favor